# Função Print parte 1

In [1]:
print ("Esta é uma mensagem exemplo") # imprime uma mensagem

Esta é uma mensagem exemplo


In [2]:
x  = 26
y = 58

print(x,y)


'''O print existe para estabelecer uma comunicação entre o programa 
e o usuario'''

26 58


In [3]:
print("Valor de  x = ", x, "e valor de y = ", y)

Valor de  x =  26 e valor de y =  58


In [4]:
print ("Valores : x = {0} e y = {1}".format(x,y))



'''Esta funcionando assim: O valor de (x) está sendo colocado no marcadpor {0} e o
 valor de (y) está sendo colocado no marcador {1}'''

Valores : x = 26 e y = 58


# Função Print parte 2

In [5]:
a = 12 # variável
b = 19 # variável
print (a,b) # imprimindo os objetos

12 19


In [6]:
# Usando um separador
print (a,b, sep= '-')   # (sep=) está atribuindo um separador


'''No lugar do separador podemos colocar o que achar coveniente'''

12-19


In [12]:
# Usando parametro (END)

print ("um\ndois\ntres")  # (\n) representa uma quebra de linha

um
dois
tres


In [15]:
# FIM DE LINHA

num1= 12
num2 = 26
print (num1)
print (num2, end = '')

'''(end) esta atribuindo um fim de linha'''


12
26

'(end) esta atribuindo um fim de linha'

# F - STRINGS


In [17]:
A = 14
B = 32

s1 = "Valores: A é {} e B é {}".format(A,B)

# Usando format 

Valores: A é 14 e B é 32


In [19]:



s2 = F"Valores: A é {A} e B é {B}"


print(s2)


# Usando F string 

Valores: A é 14 e B é 32


# Função Input parte 1

A função input() no Python é usada para capturar dados do usuário enquanto o programa está rodando. É como se o programa fizesse uma pergunta, e você, a pessoa que está usando o programa, precisa dar uma resposta.

In [20]:
nome = input("Qual é seu nome: ")


#Nesse exemplo, o programa vai mostrar a pergunta "Qual é o seu nome?" no terminal.

#Quando o programa chega nessa linha, ele fica esperando você digitar alguma coisa.

#O que você digitar será armazenado na variável nome. Ou seja, se você digitar "Patrícia Oliveira", a variável nome vai guardar o valor "Patrícia".

In [21]:
print (nome)

Patricia Oliveira


# Função Print parte 2

In [23]:
N = int(input("Digite um número: ")) # (int) converte para inteiro

print(N)

37


In [32]:
F = float(input("Digite um numero real : "))
print(F)

'''input("Digite um numero real : "): Mostra a mensagem "Digite um numero real :" e espera o usuário digitar um valor.
float(): Converte o que foi digitado de texto para número decimal (número real). Se você digitar, por exemplo, "31.47"
o float() transforma isso no número 31.47.
O valor convertido (por exemplo, 31.47) é então armazenado na variável F.'''


31.47


'input("Digite um numero real : "): Mostra a mensagem "Digite um numero real :" e espera o usuário digitar um valor.\nfloat(): Converte o que foi digitado de texto para número decimal (número real). Se você digitar, por exemplo, "31.47"\no float() transforma isso no número 31.47.\nO valor convertido (por exemplo, 31.47) é então armazenado na variável F.'

In [33]:
R = N + F

print(R)

68.47


O programa começa pedindo que você digite um número inteiro. A função input() exibe a mensagem "Digite um número: " e espera você digitar algo. Quando você digita 37, o programa pega esse valor, que originalmente é um texto, e usa a função int() para convertê-lo para um número inteiro. Esse número é armazenado na variável N.

O programa exibe o valor de N usando print(N). Nesse caso, o valor 37 é mostrado na tela.

Depois, o programa pede que você digite um número real (um número com casas decimais). A função input() exibe a mensagem "Digite um numero real :" e espera você digitar algo. Quando você digita 31.47, o programa pega esse valor e usa a função float() para convertê-lo de texto para um número real (decimal). Esse número é armazenado na variável F.

O programa então exibe o valor de F usando print(F). O valor 31.47 é mostrado na tela.

Em seguida, o programa faz uma soma entre o número inteiro armazenado em N (que é 37) e o número real armazenado em F (que é 31.47). O resultado dessa soma, 68.47, é armazenado na variável R.

Finalmente, o programa exibe o valor de R, que é 68.47, usando print(R).